package.json

```diff
@@ -9,7 +9,29 @@
     "@aragon/apps-voting": "2.0.0"
   },
   "devDependencies": {
-    "@aragon/cli": "^5.4.0"
+    "@aragon/cli": "^5.4.0",
+    "@aragon/test-helpers": "^1.0.1",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
+    "babel-eslint": "^10.0.1",
+    "babel-plugin-styled-components": "^1.10.0",
```

Comment:
These babel plugins shouldn't be needed here; they should already be in `apps/`

Reply:
You are right, I missed it when I merged, sorry. It's fixed in the last commit.